src/BookStore/Domain/Model/Book.php

```diff
@@ -17,31 +17,70 @@
 class Book
 {
     #[ORM\Embedded(columnPrefix: false)]
-    public BookId $id;
+    public readonly BookId $id;
 
     public function __construct(
         #[ORM\Embedded(columnPrefix: false)]
-        public BookName $name,
+        protected BookName $name,
 
         #[ORM\Embedded(columnPrefix: false)]
-        public BookDescription $description,
+        protected BookDescription $description,
 
         #[ORM\Embedded(columnPrefix: false)]
-        public Author $author,
+        protected Author $author,
 
         #[ORM\Embedded(columnPrefix: false)]
-        public BookContent $content,
+        protected BookContent $content,
 
         #[ORM\Embedded(columnPrefix: false)]
-        public Price $price,
+        protected Price $price,
```

Comment:
Private please, the only reason this class is not final is doctrine lazy proxies.

Reply:
I changed that in the last minute from `private` (just because of the class not being final) … OK, I'll change it again

Reply:
Done